lua/entities/gmod_wire_expression2/core/selfaware.lua

```diff
@@ -206,26 +206,29 @@ end
 --[[******************************************************************************]]--
 -- Extensions
 
+local getExtensionStatus = E2Lib.GetExtensionStatus
+local e2Extensions
+local e2ExtensionsTable
+-- See postinit for these getting initialized
+
 [nodiscard]
 e2function array getExtensions()
-	return table.Copy(E2Lib.GetExtensions())
+	local ret = {}
+	for k, v in ipairs(e2Extensions) do -- Optimized shallow copy
+		ret[k] = v
+	end
+	return ret
 end
 
-local getExtensionStatus = E2Lib.GetExtensionStatus
+__e2setcost(10)
```

Comment:
This should cost a lot more than `getExtensionStatus(s)` which just indexes a table, as this has to clone a table which is still a lot of work. Either way you shouldn't be calling it often. I'd say 60 ops would be quite generous considering a lambda call is 15 ops, function call is 10 ops